back/app/policies/initiative_status_policy.rb

```diff
@@ -34,7 +34,7 @@ def filter_out_review_statuses
       else
         public_codes = InitiativeStatus::NOT_REVIEW_CODES
         # We want to show only statuses of initiatives available to the current user.
-        user_codes = initiatives.joins(:initiative_status).distinct.pluck('initiative_statuses.code')
+        user_codes = initiatives.joins(:initiative_status).distinct.pluck('initiative_status.code')
```

Comment:
Fixes [this error](https://app.circleci.com/pipelines/github/CitizenLabDotCo/citizenlab/124640/workflows/e473f83c-0c64-4987-8992-c5116a8a4272/jobs/298077)

Reply:
The fix seems to be fine, but maybe that can be broken in other places too and maybe that's why joins were written so verbously `joins('LEFT OUTER JOIN`. 

I ran e2e tests in this branch and in `TAN-46-proposal-approval-feature`, let's see how many fail.